src/main/kotlin/org/misarch/catalog/event/EventPublisher.kt

```diff
@@ -0,0 +1,24 @@
+package org.misarch.catalog.event
+
+import io.dapr.client.DaprClient
+import kotlinx.coroutines.reactor.awaitSingleOrNull
+import org.springframework.stereotype.Component
+
+/**
+ * Publisher for dapr events
+ */
+@Component
+class EventPublisher(private val client: DaprClient) {
+
+    /**
+     * Publishes an event via the Dapr client.
+     * Uses the pubsub component.
+     *
+     * @param topic the topic to publish to, automatically prefixed with "/catalog"
+     * @param message the message to publish
+     */
+    suspend fun publishEvent(topic: String, message: Any) {
+        client.publishEvent("pubsub", "catalog/$topic", message).awaitSingleOrNull()
```

Comment:
Have we documented anywhere that all events must be published on `pubsub`?

Reply:
not yet
but I think I should write a docs page for the event stuff
do you think this is an ADR?
we do not really have/discussed multiple options

Reply:
That doesn't need to be an ADR, a simple mention inside the remaining dev docs is enough